### PR TITLE
Template update for nf-core/tools version 2.12

### PR DESCRIPTION
**Pull request by svarona**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/viralrecon/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/viralrecon _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
-# ![nf-core/viralrecon](docs/images/nf-core-viralrecon_logo_light.png#gh-light-mode-only) ![nf-core/viralrecon](docs/images/nf-core-viralrecon_logo_dark.png#gh-dark-mode-only)
-
+<h1>
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="docs/images/nf-core-viralrecon_logo_dark.png">
+    <img alt="nf-core/viralrecon" src="docs/images/nf-core-viralrecon_logo_light.png">
+  </picture>
+</h1>
```
Comment (maxulysse) [inline README.md:6]:
```suggestion
</h1>

```

Need an extra line not to break the badges

File: package-lock.json
Comment (maxulysse) [inline package-lock.json:1]: where is this file coming from?

File: package.json
Comment (maxulysse) [inline package.json:1]: where is this file coming from?

File: package-lock.json
Comment (svarona) [reply package-lock.json:1]: I actually don't know, I thought it came from the template but it doesn't seem to. Should I remove this one and the other one?

Comment (maxulysse) [reply package-lock.json:1]: I don't think it's from the TEMPLATE indeed, cf:
- https://github.com/nf-core/viralrecon/tree/TEMPLATE
- https://github.com/nf-core/tools/tree/dev/nf_core/pipeline-template

So I'd go for removal if you're unsure why these files are here

**Review by maxulysse — APPROVED** [review]
LGTM

**Pull request merged**